Resources/Locale/en-US/game-ticking/game-presets/preset-thief.ftl

```diff
@@ -0,0 +1,8 @@
+thief-role-greeting = 
+    You are a criminal. 
```

Comment:
```suggestion
    You were caught and have had a pacifism implant installed due to your prior criminal activity. 
```
I feel like there should really be something like this in the flavor text - pacifism is a good and creative solution to encouraging the thief to be a non-violent antagonist since it forces the player to use creative solutions, HOWEVER with the way this is worded people may see the pacifism as only a half-assed solution to making the thief not commit violence instead of a core gameplay idea.